foundation_page.py

```diff
@@ -0,0 +1,15 @@
+import time
+from utils import is_images_equal
+
+class FoundationPage():
+    def wait_page_stable(self, driver):
```

Comment:
You don’t need driver as argument here, store driver as class field in init and then use with self